gradle/utils.gradle

```diff
@@ -12,6 +12,52 @@ ext.configureAndroidApp = { project ->
     configureAndroidModule(project)
 }
 
+ext.configureDetektForAllSubprojects = { Project project ->
+    project.subprojects {
+        afterEvaluate { subProject ->
+            if (subProject.plugins.hasPlugin("kotlin") || subProject.plugins.hasPlugin("kotlin-android")) {
+                subProject.apply plugin: "io.gitlab.arturbosch.detekt"
+
+                subProject.detekt {
+                    parallel = true
+                    input = files("${subProject.projectDir}/src/")
+                    config = files("${project.rootDir}/detekt.yml")
+
+                    def baselineFile = file("${subProject.projectDir}/detekt-baseline.xml")
+                    baseline = baselineFile
+                }
+            }
+        }
+    }
+}
+
+ext.junitTestImplementation = { project ->
+    project.dependencies {
+        testImplementation deps.test.junitApi
+        testImplementation deps.test.junitParams
+        testImplementation deps.test.junitEngine
+        testImplementation deps.test.junitVintage
+    }
+}
+
+ext.junitAndroidTestImplementation = { project ->
```

Comment:
But Android does not support JUnit 5 without additional plugins. Does it even work?